docs/explanation/gnovm.md

```diff
@@ -0,0 +1,27 @@
+---
+id: gnovm
+---
+
+# GnoVM
+
+GnoVM is a virtual machine that interprets Gnolang, a custom version of Golang optimized for blockchains, featuring automatic state management, full determinism, and idiomatic Go.
+It works with Tendermint2 and enables smarter, more modular, and transparent appchains with embedded smart-contracts.
+It can be used in TendermintCore, forks, and non-Cosmos blockchains.
+
+Read the ["Intro to Gnoland"](https://staging.gno.land/r/gnoland/blog:p/intro) blogpost.
```

Comment:
```suggestion
Read the ["Intro to Gnoland"](https://test3.gno.land/r/gnoland/blog:p/intro) blogpost.
```

staging.gno.land -> 404

post portal loop; we can use https://gno.land/r/gnoland/blog:p/intro.